dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/VertexAI/VertexAIGeminiTextGenerationClient.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Connectors.GoogleVertexAI;
+
+/// <summary>
+/// Represents a client for interacting with the text generation gemini models by Vertex AI.
+/// </summary>
+// todo: remove this class when gemini vertex ai (preview api) support non-streaming chat completion
+internal sealed class VertexAIGeminiTextGenerationClient : GeminiTextGenerationClient, IGeminiTextGenerationClient
```

Comment:
This is a empty class for now, if there's more code to be added, that's fine, if not may consider removing it.